settings.yml

```diff
@@ -25,3 +25,11 @@ zuul:
     - periodic-sanity
     - experimental
     - experimental-provision
+zuul_connector:
+  host: '10.10.10.5'
+  username: 'vagrant'
+  user_key_file: '~/.vagrant.d/insecure_private_key'
+  host_key_file: 'acid/features/zuul_connector/tests/static/host_key.pub'
```

Comment:
Why we are using `/test/static` directory for development files which looks not good. Besides that when we are in test or development environment this is irrelevant because in these environments connector will use AutoAddPolicy policy.
This line should be commented out and leave here only as example or set to `/tmp/acid_host_key.pub`